docs/core/testing.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Testing
+order: 6
+---
+
+<a name="testing"></a>
```

Comment:
This `<a>` tag isn't needed anymore btw. Also either once `basics-for-docs` is merged (or we can rebase) this is the file where this page should go: https://github.com/FormidableLabs/urql/blob/basics-for-docs/docs/advanced/testing.md

Reply:
```suggestion
```